packages/angular/.storybook/manager-head.html

```diff
@@ -1,45 +1,22 @@
-<meta
-	name="keywords"
-	content="IBM, design, system, Carbon, design system, data visualization, charts, Bluemix, styleguide, style, guide, components, library, pattern, kit, component, cloud, angular"
-/>
+<meta name="keywords"
+	content="IBM, design, system, Carbon, design system, data visualization, charts, Bluemix, styleguide, style, guide, components, library, pattern, kit, component, cloud, angular" />
 
-<meta
-	name="description"
-	content="This is the core Angular version of Carbon Charts, an implementation of Carbon Design System's data-visualization patterns."
-/>
+<meta name="description"
+	content="This is the core Angular version of Carbon Charts, an implementation of Carbon Design System's data-visualization patterns." />
 
 <!-- Open Graph -->
 <meta property="og:title" content="Carbon Charts - Angular" />
 <meta property="og:site_name" content="Carbon Charts" />
-<meta
-	property="og:description"
-	content="This is the Angular version of Carbon Charts, an implementation of Carbon Design System's data-visualization patterns."
-/>
-<meta
-	property="og:image"
-	content="https://charts.carbondesignsystem.com/angular/share.png"
-/>
-<meta
-	property="og:url"
-	content="http://charts.carbondesignsystem.com/angular"
-/>
+<meta property="og:description"
+	content="This is the Angular version of Carbon Charts, an implementation of Carbon Design System's data-visualization patterns." />
+<meta property="og:image" content="https://charts.carbondesignsystem.com/angular/share.png" />
+<meta property="og:url" content="http://charts.carbondesignsystem.com/angular" />
 
 <!-- Favicon -->
 <link rel="icon" href="logo.svg" />
 
 <!-- Social -->
 <meta name="twitter:card" content="summary_large_image" />
 <meta name="twitter:image:alt" content="Carbon Charts - Angular" />
-<meta
-	name="twitter:image"
-	content="https://charts.carbondesignsystem.com/angular/share.png"
-/>
-<meta name="twitter:site" content="@_carbondesign" />
-
-<!-- Storybook override -->
-<script>
-	setTimeout(function () {
-		document.querySelector('div.sidebar-header > div > a').innerHTML =
-			'Carbon Charts - Angular';
-	}, 1000);
-</script>
+<meta name="twitter:image" content="https://charts.carbondesignsystem.com/angular/share.png" />
+<meta name="twitter:site" content="@_carbondesign" />
```

Comment:
```suggestion
<meta name="twitter:site" content="@_carbondesign" />

```

Reply:
Also an issue of prettier automatically stripping final CR or LFs from JSON and HTML files.